gems/aws-sdk-core/lib/aws-sdk-core/util.rb

```diff
@@ -95,11 +95,11 @@ def deserialize_number(str)
       def deserialize_time(value)
         case value
         when nil then nil
-        when /^\d+$/ then Time.at(value.to_i)
+        when /^\d+$/ then Time.at(value.to_i).utc
```

Comment:
Do we have any specs for this?  I can't seem to find any.  I guess it is covered by protocol tests, but I think probably worth adding a specific spec for this method.